### PR TITLE
Merge Branch-AboutUs-Vinod with master

### PR DESCRIPTION
**Pull request by Vinodjayakumar124**
Closes #8

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -48,12 +48,12 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
 
-### James Doe
+### Vinod s/o Jaya Kumar
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/Vinod.png" width="200px">
 
-[[github](http://github.com/johndoe)]
+[[github](http://github.com/Vinodjayakumar124)]
 [[portfolio](team/johndoe.md)]
```
Comment (Qiaoran-M) [inline docs/AboutUs.md]: perhaps you can change the portfolio file name.

**Review by Qiaoran-M — COMMENTED** [review]
LGTM except for minor issue 👍

File: docs/AboutUs.md (hunk 1 above)
Comment (Vinodjayakumar124) [reply docs/AboutUs.md]: Ah yes I hadn't created a portfolio page at that moment so I left the placeholder. Have made the necessary changes now. Thanks!

**Review by tanhl2000 — APPROVED** [review]
Looks good

File: docs/AboutUs.md [hunk 2]
```diff
@@ -48,12 +48,12 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 * Role: Developer
 * Responsibilities: Dev Ops + Threading
 
-### James Doe
+### Vinod s/o Jaya Kumar
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/Vinod.png" width="200px">
```
Comment (Qiaoran-M) [inline docs/AboutUs.md]: I think the image and portfolio file name should be your GitHub username in all lowercase as mentioned [here](https://nus-cs2103-ay2223s1.github.io/website/schedule/week7/project.html#2-update-project-website-aboutus-readme)

Comment (Vinodjayakumar124) [reply docs/AboutUs.md]: Thank you for catching this

File: docs/AboutUs.md [hunk 3]
```diff
@@ -48,12 +48,12 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
-### James Doe
+### Vinod s/o Jaya Kumar
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/vinod.png" width="200px">
 
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](http://github.com/Vinodjayakumar124)]
+[[portfolio](team/Vinodjayakumar124.md)]
```
Comment (Qiaoran-M) [inline docs/AboutUs.md:57]: you need to change the portfolio file name also, `[[portfolio](team/Vinodjayakumar124.md)]` to `[[portfolio](team/vinodjayakumar124.md)]`, I forgot to mention this in my previous review.
截屏2022-09-29 下午5 50 06

Comment (Vinodjayakumar124) [reply docs/AboutUs.md:57]: I'm sry I missed this out. Have edited it.

**Pull request merged**